frontend/src/bundles/common/components/radio/radio.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+    type Control,
+    type FieldPath,
+    type FieldValues,
+} from 'react-hook-form';
+
+import { useFormController } from '../../hooks/hooks.js';
+
```

Comment:
I think it would be better to use #
`import { useFormController } from '~/bundles/common/hooks/hooks.js';`

Reply:
Thanks for noticing! I fixed